### ISSUE TITLE
[BUG]:  CPU cost many time !!!

### ISSUE THREAD

**Issue by Johnson-yue** [hpcaitech/ColossalAI#2026]
### 🐛 Describe the bug

run ColossalAI/example/image/diffusion    and install environment done.
then I test train my own dataset with ldm and colossal

it very very slow, and the CPU time is very huge.
![image](https://user-images.githubusercontent.com/10268274/203932397-3cc82e8f-c98b-4730-ba21-1a605635f6b3.png)

### Environment

CUDA: 11.7
Python: 3.9.5
Pytorch:  1.11.0 torchvision：0.12.0
pytorch-lightning： 1.8.1
colossalAI： 0.1.10+torch1.11cu11.3

**Comment by Johnson-yue** [comment]
my config is :
`model:
  base_learning_rate: 1.0e-04
  target: ldm.models.diffusion.ddpm.LatenDiffusion
  params:
    linear_start: 0.00085
    linear_end: 0.0120
    num_timesteps_cond: 1
    log_every_t: 200
    timesteps: 1000
    first_stage_key: image
    cond_stage_key: class_label
    image_size: 32
    channels: 4
    cond_stage_trainable: true   # Note: different from the one we trained before
    conditioning_key: crossattn
    monitor: 'train/loss'
    scale_factor: 0.18215
    use_ema: False
    classifier_free: True 

    scheduler_config: # 10000 warmup steps
      target: ldm.lr_scheduler.LambdaLinearScheduler
      params:
        warm_up_steps: [ 1 ] # NOTE for resuming. use 10000 if starting from scratch
        cycle_lengths: [ 10000000000000 ] # incredibly large number to prevent corner cases
        f_start: [ 1.e-6 ]
        f_max: [ 1.e-4 ]
        f_min: [ 1.e-10 ]

    unet_config:
      target: ldm.modules.diffusionmodules.openaimodel.UNetModel
      params:
        image_size: 32 # unused
        in_channels: 4
        out_channels: 4
        model_channels: 128
        attention_resolutions: [ 8, 4, 2, 1 ]
        num_res_blocks: 2
        channel_mult: [ 1, 2,2, 4, 4 ]
        num_heads: 8
        use_spatial_transformer: True
        transformer_depth: 1
        context_dim: 512
        use_checkpoint: False
        legacy: False

    first_stage_config:
      target: ldm.models.autoencoder.AutoencoderKL
      params:
        embed_dim: 4
        monitor: val/rec_loss
        ddconfig:
          double_z: true
          z_channels: 4
          resolution: 256
          in_channels: 1
          out_ch: 1
          ch: 32
          ch_mult:
          - 1
          - 2
          - 4
          - 8
          num_res_blocks: 2
          attn_resolutions: []
          dropout: 0.0
        lossconfig:
          target: torch.nn.Identity

lightning:
  trainer:
    accelerator: 'gpu' 
    devices: 1
    log_gpu_memory: all
    max_epochs: 2
    precision: 16
    auto_select_gpus: False
    strategy:
      target: lightning.pytorch.strategies.ColossalAIStrategy
      params:
        use_chunk: False
        enable_distributed_storage: True,
        placement_policy: cuda
        force_outputs_fp32: False

    log_every_n_steps: 2
    logger: True
    default_root_dir: "/tmp/diff_log/"
    profiler: pytorch
`


I remove the data and second_stage info, because it is not necessary

**Comment by Fazziekey** [comment]
Thanks for your issue,I will check this problem

**Comment by Fazziekey** [comment]
our new Gemini fix this problem for Cpu, you can install the latest version of ColossalAI
